src/gwt/src/org/rstudio/studio/client/workbench/views/connections/ui/NewConnectionSnippetDialog.css

```diff
@@ -0,0 +1,33 @@
+@CHARSET "UTF-8";
+
+@external gwt-Label;
```

Comment:
Nit: These lines are unnecessary unless you actually reference the `gwt-Label` class in your CSS (it just tells GWT not to obfuscate the class name when it sees it later)